svf-llvm/lib/SymbolTableBuilder.cpp

```diff
@@ -590,8 +592,8 @@ ObjTypeInfo* SymbolTableBuilder::createObjTypeInfo(const Value* val)
     // (2) Other objects (e.g., alloca, global, etc.)
     else
     {
-        if(const PointerType* refTy = SVFUtil::dyn_cast<PointerType>(val->getType()))
-            objTy = getPtrElementType(refTy);
+        if(SVFUtil::isa<PointerType>(val->getType()))
```

Comment:
It looks to me that this pointee type has to be inferred if this val is an object from a malloc site.

Reply:
> It looks to me that this pointee type has to be inferred if this val is an object from a malloc site.

This branch cannot be a malloc site. The object is alloca, global, etc. The malloc site is not yet done (to be done in inferTypeOfHeapObjOrStaticObj).

Reply:
I see.